### ISSUE TITLE
Glacial Tidestorm missing

### ISSUE THREAD

**Issue by Hemario** [SacredDuckwhale/Rarity#167]
Just did my first mythic Jaina run and I notice that Glacial Tidestorm mount in Rarity is just not showing up.

**Comment by SacredDuckwhale** [comment]
This mount is not currently being tracked, as it's a guaranteed mount. I could add it to the database, but making sure the tracking actually works would probably be tricky since it would be far too time-consuming for me to test on my own.

**Comment by petems** [comment]
I think I have this working in my rarity:

Mount - 166705 - 2/20 - Mythic Raid - Boss: Lady Jaina Proudmoore

If I open as a PR would you accept?

**Comment by SacredDuckwhale** [comment]
Sure! I just don't have time to test it myself, but if it's working for you then surely it will for others, too :)

**Comment by SacredDuckwhale** [comment]
Actually, I think the defeat detection would be the trickiest part (the system is somewhat convoluted), but if that's not working it's probably not a big deal either.

**Comment by SacredDuckwhale** [comment]
Would anyone happen to know if this mount is now a "regular" rare drop in Shadowlands?

**Comment by petems** [comment]
Yep, since SL it's back to a 1% drop, so can be moved to a regular percentage based drop
